lib/bolt/cli.rb

```diff
@@ -412,8 +425,12 @@ def validate(options)
         end
       end
 
-      if options[:nodes].empty? && options[:mode] != 'plan' && options[:action] != 'show'
-        raise Bolt::CLIError, "Option '--nodes' must be specified"
+      if options[:mode] != 'plan' && options[:action] != 'show'
```

Comment:
We didn't really address it in the ticket, so it's probably better to skip it. However I might expect `--query` to work similar to `--nodes` for a plan.

Reply:
I was considering that, but decided that since the reason for `--nodes` for plans is just uniformity during the learning experience, it doesn't seem as obvious that `--query` should behave the same.